qiita_db/support_files/patches/14.sql

```diff
@@ -0,0 +1,2 @@
+-- we don't really need an SQL patch, but having nothing here breaks the build
```

Comment:
When I had an empty file, the build failed, that's why I added those
lines.

On (Feb-09-15| 9:55), adamrp wrote:
> > @@ -0,0 +1,2 @@
> > +-- we don't really need an SQL patch, but having nothing here breaks the build
> 
> Doesn't necessarily need to change, but no SQL statements are required to be in file, AFAIK (the file just needs to exist).
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/829/files#r24348917